index.js

```diff
@@ -48,7 +48,7 @@ module.exports = function(method, database, migrate, stream, live, concurrency,
     .pipe(migrator)
       .on('error', callback)
       .on('finish', function() {
-        if (migrate.finish) migrate.finish(migrator.log, done);
+        if (migrate.finish) migrate.finish(dyno, migrator.log, done);
```

Comment:
Could we just adjust this so that the `finish` function receives a null object if it is a dry run?
```js
migrate.finish(live ? dyno : null, migrator.log, done);
```